### PR TITLE
chore(infra): cleanup npm upgrade spamm

### PR DESCRIPTION
**Pull request by brynjarorng**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/air-discount-scheme/api/infra/api.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export const serviceSetup = (services: {
         staging: 'cdn.contentful.com',
         prod: 'cdn.contentful.com',
       },
+      NO_UPDATE_NOTIFIER: 'true',
```
Comment (pshomov) [inline apps/air-discount-scheme/api/infra/api.ts:39]: Maybe set this in the global env section?

Comment (brynjarorng) [reply apps/air-discount-scheme/api/infra/api.ts:39]: Are you talking about file in `infra/src/environments.ts`? It is already set there but it did not work in every location which is really strange

Comment (pshomov) [reply apps/air-discount-scheme/api/infra/api.ts:39]: That's weird. Let's fix that issue after this fix.

**Review by pshomov — APPROVED** [review]
LGTM

**Pull request merged**